meshapp/src/index.tsx

```diff
@@ -1,15 +1,15 @@
-
 import React from "react";
 import ReactDOM from "react-dom/client";
 import "./index.css";
 import TwoFactorAuthReminders from "./two-factor-auth/two-factor-reminder";
 
 import LoggedOutPage from "./home/logged-out/home";
-
+import Slider from "./ProfileCardCarousel/Swiper";
 const root = ReactDOM.createRoot(document.getElementById("root")!);
 root.render(
   <React.StrictMode>
-    <TwoFactorAuthReminders />
+    {/* <TwoFactorAuthReminders /> */}
```

Comment:
Why is this being removed?

Reply:
I commented it out for a while just to focus on the profile cards and forgot to put it back it, I added it back